src/components/NavBar/NavBar.tsx

```diff
@@ -20,12 +20,14 @@ const NavBar = () => {
           <MainLinks />
         </Navbar.Section>
         <Navbar.Section className='justify-self-end mt-auto pb-4 mx-auto'>
-          <GenericButton
-            text='Submit'
-            onClick={openSubmissionModal}
-            disabled={isLoading}
-            loading={isLoading}
-          />
+          <Tooltip label='Submit current question'>
```

Comment:
Nitpick:
Do we want to state that clicking this moves to the next question?
> "Submit question and move to next"

Idk how to word it.

Reply:
yeah